src/tests/ftest/io/fio_small.yaml

```diff
@@ -38,6 +38,8 @@ pool:
   nvme_size: 20000000000
   svcn: 1
   control_method: dmg
+container:
```

Comment:
Is this change related, what does it do?

Reply:
Yes it is related. This is only specifying a container of type "POSIX". 
Previously we were directly running the command "daos cont create --pool=$DAOS_POOL --svc=$DAOS_SVCL --type=POSIX".
But the output of this command line seems to have got changed a bit recently hence the output which was being caught was different.
So as part of this PR we are doing the same thing but just in a bit better manner using the "daos" utility incorporated in /ftest/utils/daos_util.py

Reply:
Ah ok.  If the output of the command truly has changed and is the cause of this then that's a bug with the command isn't it?  Are you able to verify the output directly?